R/root.R

```diff
@@ -24,6 +24,16 @@
 ##' path <- withr::local_tempfile()
 ##' hipercow_init(path)
 hipercow_init <- function(root = ".", driver = NULL, ...) {
+  if ((is.null(driver)) && (length(list(...)) > 0)) {
+    args <- list(...)
+    cli::cli_alert_warning(
+      "driver was not specified, but extra args were detected: ")
+    ul <- cli::cli_ul()
```

Comment:
See comments in #134 about a better way to do these

Reply:
We decided to omit these...